lib/mountain_view/component.rb

```diff
@@ -2,6 +2,10 @@ module MountainView
   class Component
     attr_reader :name
 
+    def self.stub_example
+      File.read(File.expand_path('../example.yml',__FILE__))
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.